zulip/examples/create-user

```diff
@@ -29,7 +29,9 @@ usage = """create-user --new-email=<email address> --new-password=<password> --n
 Create a user. You must be a realm admin to use this API, and the user
 will be created in your realm.
 
-Example: create-user --site=http://localhost:9991 --user=rwbarton@example.com --new-email=jarthur@example.com --new-password=random17 --new-full-name 'J. Arthur Random' --new-short-name='jarthur'
+Example: create-user --site=http://localhost:9991 --new-email=jarthur@example.com --new-password=random17 --new-full-name 'J. Arthur Random' --new-short-name='jarthur'
+
+Specify your authentication details in a ~/.zuliprc file.
```

Comment:
I think maybe we want this to be something like 

"Specify your Zulip API credentials and server in a ~zuliprc file or using the options"

Reply:
Sure! Is `~zuliprc` some convention?

Reply:
No it's a typo :)